Tools-Override/RunnerTemplate.Unix.txt

```diff
@@ -77,8 +77,8 @@ echo Commands:
 [[TestRunCommandsEcho]]
 pushd $EXECUTION_DIR
 [[TestRunCommands]]
-popd
 test_exitcode=$?
+popd
```

Comment:
👍 Pretty sure this was it. It is getting `popd`'s exit code which always looks like a success...